XamarinCommunityToolkit/Views/GravatarImageSource/GravatarImageExtension.shared.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	[ContentProperty(nameof(Email))]
+	public class GravatarImageExtension : GravatarImageSource, IMarkupExtension<GravatarImageSource>
+	{
+		public GravatarImageSource ProvideValue(IServiceProvider serviceProvider)
+		{
+			if (serviceProvider is null)
```

Comment:
We don't use `is` to check for null, can you please change it (and others) to `==` or `!=`, to follow our code style?